resources/views/includes/footer.blade.php

```diff
@@ -5,7 +5,7 @@
             <img class="footer-item" src="https://image.flaticon.com/icons/svg/77/77196.svg">
             <a href="/create">
                 <img class="footer-item" src="https://image.flaticon.com/icons/svg/77/77173.svg">
-                <a href="#index">
+                <a href="/stock">
```

Comment:
It's better to use `{{ url('stock') }}` or even better to give your route a name and use `{{ route('stock') }}`

Reply:
added routes.